docs/testnet/tutorial.md

```diff
@@ -0,0 +1,158 @@
+# Getting Started
+In this tutorial, you build your own Obscuro dApp from the start. This is an excellent way to experience a typical development process as you learn how Obscuro dApps are built, design concepts, tools and terminology.
+
+# Let's go
+This 'Guessing Game' tutorial provides an introduction to the fundamentals of Obscuro and shows you how to:
```

Comment:
I'd reword this as, "In this tutorial, we'll build a 'Guessing Game' dApp that provides..."